app/javascript/controllers/toggle_field_controller.js

```diff
@@ -22,10 +22,11 @@ export default class extends Controller {
     const field = event.target.value
 
     this.fieldTargets.forEach((el) => {
-      if (field !== el.getAttribute("field")) {
-        el.classList.add("d-none")
-      } else {
+      const types = el.getAttribute("field").split(" ")
```

Comment:
this small change is to account for being able to handle display toggling for multiple types. in this case, `mp3`, `wav`, or `flac`.